src/source/rtl_text_plugin.js

```diff
@@ -1,11 +1,30 @@
 // @flow
 
 import { Event, Evented } from '../util/evented';
+import window from '../util/window';
 
 let pluginRequested = false;
 let pluginURL = null;
 let foregroundLoadComplete = false;
 
+const isAbsoluteUrl = (url) => {
+    // handles https, http, and protocol-relative URLs
+    const isAbsolute = new RegExp('^([a-z]+://|//)', 'i');
+    return isAbsolute.exec(url);
+};
+
+const resolvePluginUrl = (path) => {
+    if (isAbsoluteUrl(path)) {
+        return path;
+    }
```

Comment:
Do we need to special case absolute URLs? It looks like we could also just happily pipe absolute/protocol-relative URLs through the `a`/`href` resolver. If we need this, it'd be a good idea to add the absolute URL special case to the GeoJSON resolver (see below) as well)